src/stores/pacbioPoolCreate.js

```diff
@@ -652,9 +657,18 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
           data: aliquots,
           includeRelationships: true,
         })
-
         // Set the used_aliquots
         Object.values(usedAliquots).forEach((usedAliquot) => {
+          // Creates a request association for the used aliquot based on the source type
```

Comment:
I think we need a type?

Reply:
Did you mean a type for a volume? We have set different attributes in library resource in service side  and in ui it is set  as fields `available_volume` and `used_volume` in usedAliquot object.

Reply:
For aliquot. There is a fair amount of logic for validation and checking if it is for a request. We have done this with other objects. It just means that the behaviour is encapsulated and you would be able to reuse it in libraries, pools and sequencing runs.